app/models/feature_metrics.rb

```diff
@@ -0,0 +1,62 @@
+class FeatureMetrics
+  include ActionView::Helpers::NumberHelper
+
+  def average_time_to_get_references(
+    start_time,
+    end_time = Time.zone.now.beginning_of_day
+  )
+    times_to_get = time_to_get_references(start_time, end_time)
```

Comment:
When I read this i wasn't completely sure what times_to_get_references was returning. I'm struggling to come up with a better name though.

`time_to_get_references_for_each_application` seems a bit clunky.

Reply:
Actually the above wouldn't really work as it is only returning times where both references are received.

I think it's probs fine as is 👍